ngsArchiveLinker.pl

```diff
@@ -275,6 +277,35 @@ sub createLinks{
 
 }
 
+#get the files for the given file type.  Either link or download given the options
+sub loopFileType{
+	my $sample = shift;
+	my $type = shift;
+	my $download = shift;
+	my $sampleDir = shift;
+	my $duplicateLevel = shift;
+	my $client = shift;
+	my $headers = shift;
+
+	foreach my $fileref(@{$sample->{$type}}){
+		my $filename = $fileref->{file};
+		my $basename = basename($filename);
+
+		my $newfile = "$sampleDir/$basename";
+
+		if($download){
+			downloadFile($fileref->{href},$newfile,$client,$headers,$duplicateLevel,$type);
+		}
+		else{
+			if(! -e $filename){
+				die "Error: Script cannot see a file to be linked: $filename.  Ensure you have access to the sequence files directory.";
```

Comment:
Indicate **assemblies** and not just **sequence files** in error message.

Reply:
Done in e8b07dc